src/Analysis/GWASResults/Components/Diagrams/lib/pheweb_plots.js

```diff
@@ -40,6 +40,7 @@ import {memoize, property, range, some, sortBy, template} from 'lodash';
 /* eslint prefer-destructuring: 0 */ // --> OFF
 /* eslint wrap-iife: 0 */ // --> OFF
 /* eslint arrow-parens: 0 */ // --> OFF
+/* eslint no-inner-declarations: 0 */ // --> OFF
```

Comment:
why not fix the offending code, instead of adding global exceptions?

Reply:
why are you trying to keep changes to a minimum? i see that we are not maintaining this and keeping it up to date with its source

Reply:
I want to give us a chance to merge in improvements from upstream, if they ever occur. There is also a chance that I've copied a suboptimal version of PheWeb (there is more than one copy made by UMichigan), so keeping it somewhat similar to what they have will make it easier to incorporate any improvements from either version... 
There is also a chance that we'll completely diverge from their code. When this time comes, then I agree we should spend the time and take full ownership of the code, fixing all these issues.